boards/weact/stm32h5_core/weact_stm32h5_core.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: weact_stm32h5_core
+name: WeAct Studio STM32H5 Core Board
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 640
+flash: 1024
+supported:
+  - gpio
+  - can
+  - gpio
+  - uart
+  - entropy
+  - spi
+  - usb_device
+  - rtc
```

Comment:
Watchdog and can are missing

Reply:
@erwango added watchdog

Reply:
@JarmouniA now it is

Reply:
And gpio is here twice

Reply:
@gautierg-st good catch, removed one of them